framework/base/Security.php

```diff
@@ -30,6 +30,11 @@
  */
 class Security extends Component
 {
+    const DEV_URANDOM = '/dev/urandom';
```

Comment:
Yes, magic numbers are bad because both they're error prone and not descriptive. Named constants solve both issue. Too bad PHP has no private constants.